Core/include/Acts/Surfaces/BoundaryTolerance.hpp

```diff
@@ -144,31 +118,30 @@ class BoundaryTolerance {
     return BoundaryTolerance{tolerance};
   }
 
+  /// Chi2 tolerance in Cartesian coordinates
+  static auto Chi2Cartesian(const SquareMatrix2& weight,
+                            double maxChi2) noexcept {
+    Chi2CartesianParams tolerance{maxChi2, {}};
+    tolerance.weightMatrix() = weight;
+    return BoundaryTolerance{tolerance};
+  }
+
   BoundaryTolerance(const BoundaryTolerance& other) noexcept = default;
   BoundaryTolerance& operator=(const BoundaryTolerance& other) noexcept =
       default;
   BoundaryTolerance(BoundaryTolerance&& other) noexcept = default;
   BoundaryTolerance& operator=(BoundaryTolerance&& other) noexcept = default;
 
-  enum class ToleranceMode {
-    Extend,  // Extend the boundary
-    None,    // No tolerance
-    Shrink   // Shrink the boundary
+  enum class Mode {
```

Comment:
I thought just "mode" was a bit of a generic name. Do we need to change it?

Reply:
no I will revert this